tools/wpt-report-builder/formatter.mjs

```diff
@@ -217,22 +217,21 @@ function generateHtml(map, commitHash) {
         })
 
         const expand = document.querySelector('.expand');
-        let toggleState = 'collapse';
+        let expandState = 'collapse';
```

Comment:
any reason both of those states are not boolean?

Reply:
I use to change the name of button as well :sweat_smile: